innative/compile.cpp

```diff
@@ -1397,13 +1402,16 @@ void Compiler::ResolveModuleExports(const Environment* env, Module* root, llvm::
   }
 }
 
-void AddRTLibCalls(Environment* env, llvm::IRBuilder<>& builder, Compiler& mainctx)
+void AddRTLibCalls(Environment* env, llvm::IRBuilder<>& builder, Compiler& mainctx, bool isJIT)
 {
-  
 #ifdef IN_PLATFORM_WIN32
   // Internal flags for asm mem functions
   new llvm::GlobalVariable(*mainctx.mod, builder.getInt32Ty(), true, Func::ExternalLinkage, builder.getInt32(2), "__favor");
 
+  if(isJIT)
```

Comment:
Add short comment explaining that the JIT doesn't export this symbol for some reason, so we have to add it in manually.